.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+environment:
+  fast_finish: true
+  matrix:
+    - nodejs_version: "8"
```

Comment:
Single quotes.

Reply:
I think there is an `.editorconfig` setting for that